shortfin/python/shortfin_apps/sd/components/builders.py

```diff
@@ -0,0 +1,162 @@
+from iree.build import *
+import itertools
+import os
+import shortfin.array as sfnp
+
+from shortfin_apps.sd.components.config_struct import ModelParams
+
+this_dir = os.path.dirname(os.path.abspath(__file__))
+parent = os.path.dirname(this_dir)
+default_config_json = os.path.join(parent, "examples", "sdxl_config_i8.json")
+
+dtype_to_filetag = {
```

Comment:
maybe we rename the artifacts to match sfnp.Dtype attributes instead of doing little workarounds like this for old naming conventions... trying to align it